internal/cli/get.go

```diff
@@ -113,14 +113,14 @@ func (o *GetOptions) Validate(args []string) error {
 	if err != nil {
 		return err
 	}
-	if len(name) > 0 && len(o.LabelSelector) > 0 {
+	if len(name) > 0 && !strings.EqualFold(name, "summary") && len(o.LabelSelector) > 0 {
```

Comment:
This overloads the `flightctl get resource_kind/resource_name` command. Could we find an unambiguos alternative?

Maybe `flightctl get devices --summary`, although the problem with that is `get devices` implies a list of devices that then gets filtered/formatted according to the flags and the summary is a completely different thing.

Maybe instead of enhancing the get/apply/delete commands that manipulate the _API resources_ we model this as a separate, high-level command like `flightctl devices summary` or `flightctl stats devices`. Needs discussion.

Reply:
ListDevices already returns this summary, correct?  The difference is that the summary currently corresponds to the devices that it returns, which may be filtered or paginated.
Maybe we can have a new query param like "deviceSummaryOnly=true" that would ignore all the filters and just return the summary (items would be an empty list).
WDYT?

Reply:
Yes, I wouldn't say I liked it myself. Without it, we crash trying to unmarshal something not a device list. 
The challenge with the list devices API is that it fetches the devices' objects up to the limit and uses the []devices to calculate the summary on the API service. We can add another query string summaryOnly to return only the summary of the entire devices, leveraging the DB capabilities to group it without fetching the entire devices' objects. But keep in mind that the summary of the list is not reflected in the CLI, only per device: "NAME\tALIAS\tOWNER\tSYSTEM\tUPDATED\tAPPLICATIONS\tLAST SEEN." So, we need a command to reflect the summary
We should meet and discuss this.

Reply:
`GET /api/v1/devices` currently returns the following properties:
1. metadata (pagination info)
2. items (the list of devices)
3. summary (a tally of how many devices have each summary/update/application status)

The summary corresponds only to the devices returned in "items".

I propose to add a flag that will return "items" as empty.  This way, no pagination is necessary, and "summary" can easily reflect all devices (or any other group of devices, depending on the set filters).

Reply:
It will work, but what about CLI support to reflect the summary/overview?

Reply:
We can skip CLI for this IMO

Reply:
That approach makes sense to me. Need to define+document the behaviour if used in connection with `limit`, though.

Re the CLI: I'm thinking we could add a flag `--summary,-s` to print the summary after the table output (the table isn't meant to be machine-parseable anyway). And then `--summaryOnly` could only print that summary?

Reply:
Good points.  Passing any pagination, filters, etc. together with `--summaryOnly` should return 400.  We can add a flag to the CLI to print a table of the summary as you suggested.